packages/core/contracts-ovm/insured-bridge/implementation/BridgeDepositBox.sol

```diff
@@ -39,6 +39,9 @@ abstract contract BridgeDepositBox is Legacy_Testable, Legacy_Lockable {
     // ChainID of the L2 this deposit box is deployed on.
     uint256 chainId;
 
+    // Address of WETH on L1. If the deposited token maps to this L1 token then wrap ETH to WETH on the users behalf.
+    address l1Weth;
```

Comment:
For this and the `chainID` var above do we not have to explicitly set them as `public` vars?

Reply:
ye, can do.